external-crates/move/crates/move-compiler/src/to_bytecode/canonicalize_handles.rs

```diff
@@ -229,7 +229,7 @@ pub fn in_module(
     });
 }
 
-/// Reverses mapping from `StructDefinition(Index)` to `StructHandle`, so that handles for structs
+/// Reverses mapping from `StructDefinition(Index)` to `DatatpeHandle`, so that handles for structs
 /// defined in a module can be arranged in definition order.
 fn struct_definition_order(
```

Comment:
I believe this whole module needs to be updated to handle enum defs (looks like they aren't handled currently).

Reply:
This is because this does not do bytecode support,  because those names are not yet landed in main. The other diff that targeted the bytecode changes has those changes made.